### ISSUE TITLE
Compilation issues for Fedora

### ISSUE THREAD

**Issue by maxborowsky** [fossfreedom/alternative-toolbar#177]
Fedora 36 Workstation, immediately after the "clean" installation, last (0.20.2) release of plugin (**tar.gz**) an "old" method using **./autogen.sh -prefix =/usr** (which has now been replaced in README by ./configure -prefix =/usr):

```
$./autogen.sh -prefix =/usr

You should add the contents of '/usr/share/aclocal/intltool.m4' to 'aclocal.m4'.
You should update your 'aclocal.m4' by running aclocal.
configure.ac:24: warning: The macro `GLIB_GNU_GETTEXT' is obsolete.
configure.ac:24: You should run autoupdate.
(...)
configure.ac:24: warning: The macro `AC_TRY_LINK' is obsolete.
configure.ac:24: You should run autoupdate.
(...)
configure.ac:27: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:27: You should run autoupdate.
```
etc
```
$ make
configure.ac:24: warning: The macro `GLIB_GNU_GETTEXT' is obsolete.
configure.ac:24: You should run autoupdate.
(...)
configure.ac:24: warning: The macro `AC_TRY_LINK' is obsolete.
configure.ac:24: You should run autoupdate.
(...)
configure.ac:27: warning: The macro `AC_HELP_STRING' is obsolete.
configure.ac:27: You should run autoupdate.
```
etc
```
$ sudo make install
```
...... and plugin files appear in the desired directories (**/usr/lib64/rhythmbox/plugins**, **/usr/share/rhythmbox/plugins**, **/usr/share/glib-2.0/schemas**). The plugin is visible in the list of Rhythmbox plugins, its settings too, but after restarting Rhythmbox, the pre-installed list of radiostations completely disappeared from it!

Fedora 36 Workstation, an "new" method using **./configure --prefix=/usr**:

```
$ ./configure --prefix=/usr

bash: ./configure: No such file or directory
```
And that's not surprising: the archive contains the configure.ac file, but not the configure script.

I really ask the author of this good extension, which pleases all Ubuntu users, to give adequate instructions on how to compile a plugin for Fedora, and so that he does not do any unpleasant things with Rhythmbox like deleting a playlist.

**Comment by fossfreedom** [comment]
> I really ask the author of this good extension, which pleases all Ubuntu users, to give adequate instructions on how to compile a plugin for Fedora,

That responsibility needs to come from Fedora users. The only distros I develop specifically for is Debian and Ubuntu.

**Comment by luc14n0** [comment]
Hi there Max.

Have you tried this before?

``` sh
$ ./autogen.sh
$ ./configure --prefix =/usr
```

After compiling using that here, I still see the list of radio stations that comes with Rhythmbox, even after restarting it/the system.

**Comment by maxborowsky** [comment]
Hi, @luc14n0.

I do not use Fedora now, for some reason I migrated to Debian, and therefore, unfortunately, I will not be able to check your advice.

However, I note that there is still a bug with the disappearance of radio stations in the most recent version of alternative-toolbar, which is available for Debian 12.

**Comment by luc14n0** [comment]
Hmm, interesting! Taking a look at `autogen.sh` I see `test -n "$NOCONFIGURE" || "$srcdir/configure" "$@"` in the last line. So, for the average user, `./autogen.sh --prefix=/usr` should be virtually the same as I pointed above.

I'm going to double check the stock radio list before and after installing this plugin, because unless only part of the list is disappearing, it seems fine to me, in my system. Does the list get reduced, or does it get completely emptied for you?

**Comment by maxborowsky** [comment]
@luc14n0,

>**completely emptied**

Exactly. This happens immediately after installing and customizing alternative-toolbar and sometimes after minor changes in its settings (I did not understand the pattern). To restore the list, I have to use the command

`gsettings set org.gnome.rhythmbox.plugins.iradio initial-stations-loaded false`

**Comment by luc14n0** [comment]
I'll keep that in mind when doing tests.

**Comment by luc14n0** [comment]
Alright, after checking and unchecking several boxes in **Preferences**->**Plugins**->**Alternative Toolbar**->**Preferences** - and restarting Rhythmbox -, I still get the same **Radio** pre-installed list here:

![image](https://github.com/fossfreedom/alternative-toolbar/assets/31497094/62fd4802-c38a-48f1-a893-ee04547ca577)

But I'll keep trying, to see if the results change.
